product_docs/docs/edbcloud/beta/using_cluster/connecting_your_cluster/01_connecting_from_azure/03_vnet_vnet.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: VNet-VNet Example
+---
+
+These are the steps to connect using VNet-VNet connections. To walk through this example in your own environment, see the prerequisites described in [Connection Walkthrough Examples](../../connecting_your_cluster/#connection-walkthrough-examples).
+
+To use this method, you need to create [Azure VPN Gateways](https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-about-vpngateways) in each connected virtual network. See [Connect Using Azure VPN Gateway Point to Site](#connect-using-azure-vpn-gateway-point-to-site) for details about creating a virtual network gateway. 
+
+!!! Note
+    VPN gateway creation can take up to 45 minutes. 
+
+#### Step 1: Create a VPN Gateway for the EDB Cloud cluster virtual network
+
+ In the **Virtual network gateway** page, create the first VPN gateway for the EDB Cloud Cluster resource virtual network `vnet-japaneast`. Name the VPN gateway "vpng-edbcloud".
```

Comment:
The Title Casing In This line Looks Like it's Not Right. At the very least, the casing on "VPN Gateway" is not consistent in this page.

Reply:
@zozowell : Is there a risk of a naming conflict with `vpng-edbcloud`? e.g., if a customer wants to VPN gateway to two different VNets?

Reply:
I'd like to keep the sentence case for the step headings. We will check that we are doing that consistently. We plan to move to sentence case for all headings before the external GA. It is a more modern and easier to read style.

Reply:
> I'd like to keep the sentence case for the step headings. We will check that we are doing that consistently. We plan to move to sentence case for all headings before the external GA. It is a more modern and easier to read style.

I'm not referring to the heading, I'm referring to the line the comment is on. "Virtual network gateway" page presumably should be title cased, "gateway" is inconsistently cased and "Cluster" is title cased for some reason.

Reply:
@ba-edb 

>Okay, we should document this, then, right? The text references the "first VPN gateway", which implies that there can be many.

I meant : 
- the first one - vpn gateway at PG service side.
- the second one - vpn gateway at consumer side. 

I just realized this reads ambiguous ... let's remove `first` and change it to:

```
create the VPN gateway for the EDB Cloud Cluster resource virtual network `vnet-japaneast`.
```

Reply:
Removed reference to "First" and "Second".

Reply:
> > I'd like to keep the sentence case for the step headings. We will check that we are doing that consistently. We plan to move to sentence case for all headings before the external GA. It is a more modern and easier to read style.
> 
> I'm not referring to the heading, I'm referring to the line the comment is on. "Virtual network gateway" page presumably should be title cased, "gateway" is inconsistently cased and "Cluster" is title cased for some reason.


- Corrected the incorrect casing for "Cluster." Good catch.
- Not sure if "virtual network gateway" needs to be titled cased (if we are going with sentence case for step headings). So I have let that be as is for now.

Reply:
@moiznalwalla , you should be able to follow the instructions to check the case of the Virtual network gateway page title. We should use the same case when referring to the page in the docs. I don't think it is a proper noun so I don't think it needs to be capitalized within body text.